frontend/editor/src/pages/Login.tsx

```diff
@@ -15,15 +15,21 @@ export default function Login() {
       username: e.target.elements.username.value,
       password: e.target.elements.password.value,
     };
-    AuthService.authCookieLoginApiCookieLoginPost(formData).then(_ => {
+    AuthService.authCookieLoginApiCookieLoginPost(formData).then(() => {
       enqueueSnackbar("Login successfully!", {
         variant: "success",
       });
       // go to /editor/mobile
       //window.location.href = '/editor/mobile'; with refresh 
-      window.history.pushState({}, "", "/editor/mobile");
-      window.dispatchEvent(new PopStateEvent("popstate"));
-    }).catch(_ => {
+      if(window.outerWidth < 600) {
```

Comment:
Może lepiej przyciski jak w #18, albo chociaż User-Agent sprawdzaj?

Reply:
Przycisk będzie jak zmienię nazwy na to co jest w aplikacji mobilnej żeby się pokrywały z funkcjonalnościami swoimi a obecnie dodam User-Agent